.ci/Jenkinsfile_flaky

```diff
@@ -61,7 +64,9 @@ stage("Kibana Pipeline") {
 
 def getWorkerFromParams(isXpack, job, ciGroup) {
   if (!isXpack) {
-    if (job == 'firefoxSmoke') {
+    if (job == 'serverMocha') {
+      return kibanaPipeline.getPostBuildWorker('serverMocha', { runbld('.ci/run.sh', "Execute serverMocha") })
```

Comment:
`.ci/run.sh` will do all of the setup stuff (install node, extract bootstrap cache, etc), so that means you'll be doing that stuff a bunch of times in parallel.

The `kibanaPipeline.withWorkers` block above will do all of the set up stuff, so you should be able to replace this with a script that just runs `node scripts/mocha`